### PR TITLE
[DONE]pr/up&going/ch3/###scope&closures

### PR DESCRIPTION
**Pull request by Zenfeuer**
- Some minor fixes on Ch 3: Dentro de YDKJS
- Fix typo in Ch 1

**Yes, I promise I've read the [Contributions Guidelines](https://github.com/getify/You-Dont-Know-JS/blob/master/CONTRIBUTING.md)** (please feel free to remove this line).

### CODE REVIEW

File: up & going/ch3.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
-# Usted no sabe JS: En marcha
-# Chapter 3: Dentro de YDKJS
+# You Don't Know JS: Up & Going
```
Comment (Zenfeuer) [inline up & going/ch3.md]: @mballeng91 we need to take a decision about if we are going to translate this title (and the rest of them)

**Comment by Zenfeuer** [comment]
@mballeng91 ready for review!

File: up & going/ch3.md (hunk 1 above)
Comment (mballeng91) [reply up & going/ch3.md]: Sure, I beilieve is better to let titles as the original version. Thanks @Zenfeuer

**Comment by mballeng91** [comment]
Please review and take actioin over the comments in the commit. Thanks

**Comment by Zenfeuer** [comment]
@mballeng91 all comments are resolved, thanks!

File: up & going/ch3.md (hunk 1 above)
Comment (Zenfeuer) [reply up & going/ch3.md]: Ok @mballeng91

**Comment by mballeng91** [comment]
We're good to merge. Thanks!

**Pull request merged**